media/editor/dataDisplayContext.tsx

```diff
@@ -239,6 +252,20 @@ export class DisplayContext {
 
 	}
 
+	public countSelections(): number {
```

Comment:
Probably could be optimized with some cleverness, but this works for now...

Reply:
yeah, the part that made this tricky is the behavior where duplicate items in the selection list effectively "toggle" the selectedness of that bit. if it could be converted to a simple "flat" Set (where each byte is either in the list once or not in the list indicates if it is selected or not) that might make it a lot easier to perform these calculations